src/ScottPlot5/ScottPlot5/Primitives/Color.cs

```diff
@@ -359,4 +359,9 @@ public static Color RandomHue()
         float luminosity = 0.5f;
         return Color.FromHSL(hue, saturation, luminosity);
     }
+
+    public static System.Drawing.Color ConvertColor(ScottPlot.Color color)
```

Comment:
Perhaps it is better to change the method name to `ToColor`, similar to `FromColor`?

Reply:
i like it.  updated.

Reply:
Hi @kebox7! If you can solve the conversation it can later be merged ;)